app.js

```diff
@@ -31,23 +30,44 @@ var click_handler = function(event) {
     reset_current();

```

Comment:
there is an issue with your click handler. You never set up a listener on the page. As such, there is no way for it to actually handle events on the page. 
The basic flow for listening for an event and then handling it is
1. Define a variable and assign it a value of the DOM element you are targeting (in your case the form)
2. Define the handler function (in your case click_handler)
3. Add an event listener to the targeted DOM element (in your case the form) and pass it the handler function